### PR TITLE
added bulk upload package

### PR DESCRIPTION
**Pull request by prateek-webkul**
**BUGS:**

>Please describe the issue that you solved if its not filed.

>Otherwise please mention issue #id and use comma if your PR
>solves multiple issues.

**For things other than bugs:**

> Describe that thing in very short line, word limit is 200.
> Otherwise use **issue #id** if the issue was filed as **feature** request.

### CODE REVIEW

File: packages/Webkul/Bulkupload/src/DataGrids/Admin/ProfileDataGrid.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        $this->addColumn([
+            'index' => 'name',
+            'label' => trans('bulkupload::app.shop.bulk-upload.attribute-set-name'),
+            'type' => 'string',
+            'searchable' => true,
+            'sortable' => true,
+            'filterable' => true
+        ]);
+
+        $this->addColumn([
```
Comment (jitendra-webkul) [inline packages/Webkul/Bulkupload/src/DataGrids/Admin/ProfileDataGrid.php:54]: Arrow should be in one line

File: packages/Webkul/Bulkupload/src/DataGrids/Admin/ProfileDataGrid.php [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+{
+    /**
+     * @var integer
+     */
+    protected $index = 'id';
+
+    public function prepareQueryBuilder()
+    {
+        $queryBuilder = DB::table('bulkupload_dataflowprofile')
+        ->leftJoin('attribute_families', 'bulkupload_dataflowprofile.attribute_family_id', '=', 'attribute_families.id')
```
Comment (jitendra-webkul) [inline packages/Webkul/Bulkupload/src/DataGrids/Admin/ProfileDataGrid.php:22]: Please use tab here

File: packages/Webkul/Bulkupload/src/Database/Migrations/2019_04_23_115227_create_bulkupload_dataflowprofile_table.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+class CreateBulkuploadDataflowprofileTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('bulkupload_dataflowprofile', function (Blueprint $table) {
```
Comment (jitendra-webkul) [inline packages/Webkul/Bulkupload/src/Database/Migrations/2019_04_23_115227_create_bulkupload_dataflowprofile_table.php:16]: Table name should be bulkupload_data_flow_profiles

File: packages/Webkul/Bulkupload/src/DataGrids/Admin/ProfileDataGrid.php (hunk 1 above)
Comment (prateek-webkul) [reply packages/Webkul/Bulkupload/src/DataGrids/Admin/ProfileDataGrid.php:53]: done

File: packages/Webkul/Bulkupload/src/DataGrids/Admin/ProfileDataGrid.php (hunk 2 above)
Comment (prateek-webkul) [reply packages/Webkul/Bulkupload/src/DataGrids/Admin/ProfileDataGrid.php]: done

File: packages/Webkul/Bulkupload/src/Database/Migrations/2019_04_23_115227_create_bulkupload_dataflowprofile_table.php (hunk 3 above)
Comment (prateek-webkul) [reply packages/Webkul/Bulkupload/src/Database/Migrations/2019_04_23_115227_create_bulkupload_dataflowprofile_table.php]: done

**Comment by irisda** [comment]
Is any update for this ticket please???

File: packages/Webkul/Bulkupload/src/Resources/views/admin/bulk-upload/data-flow-profile/index.blade.php [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+@section('page_title')
+    {{ __('bulkupload::app.admin.bulk-upload.run-profile') }}
+@stop
+
+@section('content')
+
+   <!-- Import New products -->
+    <div class="import-new-products mt-45">
+        <div class="heading">
+            <h1>Add Profile</h1>
```
Comment (cagartner) [inline packages/Webkul/Bulkupload/src/Resources/views/admin/bulk-upload/data-flow-profile/index.blade.php:12]: String not translated

**Pull request closed without merging**